src/bootstrap/native.rs

```diff
@@ -175,6 +175,17 @@ impl Step for Llvm {
             cfg.define("LLVM_ENABLE_ZLIB", "OFF");
         }
 
+        // Are we compiling for iOS/tvOS?
+        if target.contains("apple") && !target.contains("darwin") {
```

Comment:
I think it would be better to explicitly check for `ios` or `tvos` like you do later on in this file.

Reply:
Resolved by https://github.com/rust-lang/rust/pull/73903/commits/67b162f043e7e52e71b7ae8bbbdb0a732ec85eb1